src/main/java/frc/robot/subsystems/chassis/Elevator.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.chassis;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Elevator extends SubsystemBase {
+  private CANSparkMax ElevNeoMotor1;
+  private CANSparkMax ElevNeoMotor2;
+
+  /** Creates a new Elevator. */
+  public Elevator() {
+    ElevNeoMotor1 = new CANSparkMax(11, MotorType.kBrushless);
+    ElevNeoMotor2 = new CANSparkMax(12, MotorType.kBrushless);
+
+    ElevNeoMotor2.follow(ElevNeoMotor1);
+
+    ElevNeoMotor2.setInverted(true);
+  }
+
+  public void SetSpeed(double speed){
+    ElevNeoMotor1.set(speed);
+  }
+
+  public Command setElevatorSpeedUp(double speed){
+    return this.runOnce(() -> ElevNeoMotor1.set(speed));
+  }
+
+  public Command setElevatorSpeedDown(double speed){
+    return this.runOnce(() -> ElevNeoMotor1.set(speed * -1));
+  }
+
+  public Command stopElevator(){
+    return this.runOnce(() -> ElevNeoMotor1.set(0));
+  }
```

Comment:
Combine the stop command into up / down commands with `this.startEnd` instead of `this.runOnce`